### PR TITLE
changed bgp configuration with examples

### PR DESCRIPTION
**Pull request by robric**
peer-group vs profiles ?

### CODE REVIEW

File: yang/ac-svc-tree-RR-revision.txt [hunk 1]
```diff
@@ -176,19 +176,43 @@ module: ietf-ac-svc
 // We also provide a placeholder to be populated with adequate parameters. The logic of the policy is 
 // Do we want authentication here ?
 //
+// + Adding local-address (i.e PE address), in case we have a loopback on the PE that must be learnt by the customer orchestrator ??
```
Comment (boucadair) [inline yang/ac-svc-tree-RR-revision.txt:179]: Good point. The address should be a read-only then.

File: yang/ac-svc-tree-RR-revision.txt [hunk 2]
```diff
@@ -176,19 +176,43 @@ module: ietf-ac-svc
...
+/// Question: should have both a profile (policy) and peer-group (more a config) 
+///
+/// example: profile + peer-group version 
+///
      |     +--rw bgp
```
Comment (boucadair) [inline yang/ac-svc-tree-RR-revision.txt:184]: FWIW, the current version we have include the following at profile level

```
`     |  |     +--rw bgp
     |  |     |  +--rw peer-groups
     |  |     |     +--rw peer-group* [name]
     |  |     |        +--rw name              string
     |  |     |        +--rw peer-as           inet:as-number
     |  |     |        +--rw address-family?   identityref`
```

File: yang/ac-svc-tree-RR-revision.txt [hunk 3]
```diff
@@ -176,19 +176,43 @@ module: ietf-ac-svc
...
-     |     |  +--rw peer-group*                  string
-     |     |  +--rw peer-as           inet:as-number
-     |     |  +--rw address-family?   identityref
+     |     |  +--rw peer-group*          string
```
Comment (boucadair) [inline yang/ac-svc-tree-RR-revision.txt:185]: How to reference a peer-group instance?

File: yang/ac-svc-tree-RR-revision.txt [hunk 4]
```diff
@@ -176,19 +176,43 @@ module: ietf-ac-svc
...
+     |     |     +--rw peer-as           inet:as-number
+     |     |     +--rw address-family?   identityref
+     |     |     +--rw local-address     string
+     |     |     +--rw bgp-neighbor-profile?        string
```
Comment (boucadair) [inline yang/ac-svc-tree-RR-revision.txt:189]: I'm afraid the neighbor-profile needs to be further defined

File: yang/ac-svc-tree-RR-revision.txt [hunk 5]
```diff
@@ -176,19 +176,43 @@ module: ietf-ac-svc
...
+     |     |     +--rw local-address?     string
+     |     |     +--rw bgp-neighbor-profile?        string
+     |     |     +--rw status
+     |     |        +--rw admin-status
+     |     |        |  +--rw status?        identityref
+     |     |        |  +--rw last-change?   yang:date-and-time
+     |     |        +--ro oper-status
+     |     |           +--ro status?        identityref
+     |     |           +--ro last-change?   yang:date-and-time
+
```
Comment (boucadair) [inline yang/ac-svc-tree-RR-revision.txt:215]: To ease identifying required changes, the structure we currently have in the yang file is provided below: 

```
        |     +--rw bgp
        |     |  +--rw peer-groups
        |     |  |  +--rw peer-group* [name]
        |     |  |     +--rw name              string
        |     |  |     +--rw peer-as           inet:as-number
        |     |  |     +--rw address-family?   identityref
        |     |  +--rw neighbor* [remote-address]
        |     |     +--rw remote-address    inet:ip-address
        |     |     +--rw peer-group?       -> ../../peer-groups/peer-group/name
        |     |     +--rw peer-as           inet:as-number
        |     |     +--rw address-family?   identityref
        |     |     +--rw status
        |     |        +--rw admin-status
        |     |        |  +--rw status?        identityref
        |     |        |  +--rw last-change?   yang:date-and-time
        |     |        +--ro oper-status
        |     |           +--ro status?        identityref
        |     |           +--ro last-change?   yang:date-and-time
```

**Review by boucadair — APPROVED** [review]

File: yang/ac-svc-tree-RR-revision.txt (hunk 1 above)
Comment (robric) [reply yang/ac-svc-tree-RR-revision.txt:179]: 100% agree !

Comment (boucadair) [reply yang/ac-svc-tree-RR-revision.txt:179]: This is now reflected in the YANG module.

**Pull request merged**